velox/vector/arrow/Bridge.cpp

```diff
@@ -88,12 +99,15 @@ class VeloxToArrowBridgeHolder {
   }
 
  private:
+  // Holds the count of total buffers
+  size_t num_buffers_ = kMaxBuffers;
+
   // Holds the pointers to the arrow buffers.
-  const void* buffers_[kMaxBuffers];
+  std::vector<const void*> buffers_;
```

Comment:
maybe just construct with the size you need, instead of explicitly doing that in the constructor?

```
std::vector<const void*> buffers_{numBuffers_};
```

Reply:
Done